entries/h/huk24.de.json

```diff
@@ -0,0 +1,19 @@
+{
+  "HUK24": {
+    "domain": "huk24.de",
+    "contact": {
+      "email": "info@huk24.de",
+      "language": "de"
+    },

```

Comment:
As the site already supports 2FA there's no reason for contact information to be stored :wink: 
```suggeston
```